bin/apostrophe

```diff
@@ -2,50 +2,62 @@
 
 require('shelljs/global');
 require('colors');
-var program = require('commander');
+var { program } = require('commander');
```

Comment:
Stomp `var` with lasers from space pls

Reply:
... It's OK to tell me that's coming later. It's not a blocker.

Reply:
I did that and walked it back so the PR wouldn't be so intense. I can certainly do that though.